harry-core/src/harry/visitors/MutatingVisitor.java

```diff
@@ -66,17 +67,52 @@ public MutatingVisitor(Run run,
         super(run, visitExecutor);
     }
 
+    public static class LtsTrackingVisitExecutor extends MutatingVisitExecutor
+    {
+        public LtsTrackingVisitExecutor(OpSelectors.DescriptorSelector descriptorSelector, DataTracker tracker, SystemUnderTest sut, SchemaSpec schema, OperationExecutor rowVisitor, SystemUnderTest.ConsistencyLevel consistencyLevel)
+        {
+            super(descriptorSelector, tracker, sut, schema, rowVisitor, consistencyLevel);
+        }
+
+        @Override
+        public void afterLts(long lts, long pd)
+        {
+            if (hadVisibleVisit)
+            {
+                StringBuilder sb = new StringBuilder();
+                sb.append("UPDATE ").append(schema.keyspace).append(".").append(schema.table)
+                  .append(" SET visited_lts = visited_lts + [").append(lts).append("] ")
```

Comment:
So we're guaranteed the schema has `visited_lts` here?

Reply:
added assert in constructor.